docs/release/feature-stages.md

```diff
@@ -0,0 +1,83 @@
+* Full documentation on user facing channels (kubeflow.org and reference websites).
+
+### Stable
+All of the guarantees for Beta and:
+* No breaking changes without a major version bump.
+* The software is recommended for all uses.
+
+## Stable Features
+
+* [SDK DSL](https://github.com/kubeflow/pipelines/tree/master/sdk/python/kfp/dsl) for constructing a pipeline.
+* [ComponentSpec](https://github.com/kubeflow/pipelines/blob/release-1.0/sdk/python/kfp/components/structures/components.json_schema.json).
+
+Note, these packages are stable in general, but specific classes, methods, and arguments might be in a different stage. Refer to [their own documentation](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.dsl.html).
+
+### Core APIs
+
+|                  | Upload | Create | Get | List | Delete | Archive | Unarchive | Enable | Disable | Terminate | Retry |
+|:----------------:|:------:|:------:|:---:|:----:|:------:|:-------:|:---------:|:------:|:-------:|:---------:|:-----:|
+| Pipeline         | ☑️  |  ☑️ |☑️|☑️ | ☑️  |         |           |
+| Pipeline Version | ☑️  |  ☑️ |☑️|☑️ | ☑️  |         |           |
+| Run              |        |  ☑️ |☑️|☑️ | ☑️  | ☑️   | ☑️     |        |         | ☑️     | ☑️ |
+| Experiment       |        |  ☑️ |☑️|☑️ | ☑️  | ☑️   | ☑️     |
+
+* Refer to the Kubeflow Pipelines [introduction]((https://www.kubeflow.org/docs/pipelines/overview/)) for
+  - An [overview](https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/) and [main concepts](https://www.kubeflow.org/docs/pipelines/overview/concepts/), for example, what is a pipeline or run in Kubeflow Pipelines.
+  - How to manage the Kubeflow Pipelines resources via the the [Kubeflow Pipelines UI](https://www.kubeflow.org/docs/pipelines/overview/interfaces/).
+* Refer to the Kubeflow Pipelines [SDK samples and tutorials](https://www.kubeflow.org/docs/pipelines/tutorials/sdk-examples/)
+for how to manage the Kubeflow Pipelines resources via the Kubeflow Pipelines client.
+* A complete API method list and a detailed description of parameters to those API methods are available at the Kubeflow Pipelines [API reference](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/).
+
+## Features in Beta
+
+* Job API
+
+  |                  | Upload | Create | Get | List | Delete | Archive | Unarchive | Enable | Disable | Terminate | Retry |
+  |:----------------:|:------:|:------:|:---:|:----:|:------:|:-------:|:---------:|:------:|:-------:|:---------:|:-----:|
+  | Job              |        |  ☑️ |☑️|☑️ | ☑️  |         |           | ☑️  | ☑️   |
+
+
+* [Upgrade support for the Kubeflow Pipelines standalone deployment](https://www.kubeflow.org/docs/pipelines/installation/standalone-deployment/#upgrading-kubeflow-pipelines).
+
+* [Built-in Visualizations](https://www.kubeflow.org/docs/pipelines/sdk/output-viewer/).
```

Comment:
For the next three list items, please revise to use sentence case.

Built-in visualizations
Pipeline metrics
Multi-user isolation